Assets/Scripts/Controllers/CameraController.cs

```diff
@@ -44,4 +62,83 @@ public void ChangeZoom(float amount)
     {
         zoomTarget = Camera.main.orthographicSize - (Settings.GetSettingAsFloat("ZoomSensitivity", 3) * (Camera.main.orthographicSize * amount));
     }
+
+    public void ChangeLayer(int newLayer) 
+    {
+        if (layerCameras != null && newLayer >= 0 && newLayer < layerCameras.Length)
+        {
+            currentLayer = newLayer;
+            for (int i = 0; i < layerCameras.Length; i++)
+            {
+                if (i < newLayer)
+                {
+                    layerCameras[i].gameObject.SetActive(false);
+                }
+                else
+                {
+                    layerCameras[i].gameObject.SetActive(true);
+                }
```

Comment:
this
```
                if (i < newLayer)
                {
                    layerCameras[i].gameObject.SetActive(false);
                }
                else
                {
                    layerCameras[i].gameObject.SetActive(true);
                }
```
could be written as this
`layerCameras[i].gameObject.SetActive(i >= newLayer);`